tmt/steps/provision/podman.py

```diff
@@ -475,11 +474,6 @@ def go(self, *, logger: Optional[tmt.log.Logger] = None) -> None:
         self._guest.start()
         self._guest.setup()
 
-        # TODO: this might be allowed with `--privileged`...
-        self._guest.facts.capabilities[GuestCapability.SYSLOG_ACTION_READ_ALL] = False
-        # ... while this seems to be forbidden completely.
-        self._guest.facts.capabilities[GuestCapability.SYSLOG_ACTION_READ_CLEAR] = False
```

Comment:
I'm not sure we should give up the distinction between these two "capabilities". Yes, it's a single capability, `CAP_SYSLOG`, but what I tried to emulate by hijacking the capabilities enum is the different level of functionality allowed: a privileged container seems to have a limited console ring buffer access, and it is possible to read the messages, but not remove them. That is something I would like to preserve, it would give users with container if not a clear picture of "this is what happened while the test was running", but at least a "this is all that has happened, sorry, can't do better".

I think it's no longer practical to squeeze this into `GuestCapabilities`, but having a way of tracking the "can read" vs "can read and erase" flavor of `CAP_SYSLOG` would be nice. I'll try to find a test that would demonstrate this behavior.

Reply:
ok, looks like I misunderstood your chat message earlier. 
So if I understand it correctly, we could deduce this based on is-container, is-rootful, is-root / has-capability? (or something like that)